README.md

```diff
@@ -172,4 +172,4 @@ class AppExtension extends CompilerExtension implements IAssetsProvider
 * 1.0.2
 	+ Optimized dependencies
 	+ Removed unused exceptions
-	+ Optimized checking for files' modifications
\ No newline at end of file
+	+ Optimized checking for files' modifications
```

Comment:
Not sure what caused that. I've used Github editor (it was just typo, so I didn't want to clone whole project)